README.md

```diff
@@ -22,6 +22,22 @@ stable PPA, `https://launchpad.net/~juju/+archive/stable`, and can be installed
     sudo apt-add-repository ppa:juju/stable
     sudo apt-get update
     sudo apt-get install juju
+    
+Getting Golang
+--------------
+
+`juju` currently depends on Go 1.8. One of the easiest ways to install golang is 
+by [getting snap](https://snapcraft.io/docs/core/install) and then installing the snap package
```

Comment:
"getting snap" reads a little funny.
Maybe something along the lines of:

... to install golang is from a [snap](https://snapcraft.io). You may need to first install the [snap client](https://snapcraft.io/docs/core/install). Installing the golang snap package is then as easy as:

    snap install go --classic